ProcessLib/Process.h

```diff
@@ -235,8 +235,31 @@ class Process
     }
 
 protected:
+    /** This function is for general cases, in which all equations of the
+     coupled processes have the same number of unknowns. For the general cases
+     with the staggered scheme, all equations of the coupled processes share one
+     DOF table hold by \verb{_local_to_global_index_map}. Other cases can be
+     considered by overloading this member function in the
+     derived class.
+     */
     virtual void constructDofTable();
 
+    /**
+     * Construct the DOF table for a specified process in the monolithic scheme,
```

Comment:
In the monolithic case the process is not specified...

Reply:
Dropped the words.